tools/darwin/packaging/darwin_embedded/mkdeb-darwin_embedded.sh.in

```diff
@@ -9,7 +9,7 @@ SWITCH=`echo $1 | tr [A-Z] [a-z]`
 DIRNAME=`dirname $0`
 DSYM_TARGET_DIR=/Users/Shared/xbmc-depends/dSyms
 DSYM_FILENAME=@APP_NAME@.app.dSYM
-ARM64=false
+ARM64=true
```

Comment:
Is there any case where ARM64 isn't true?

Reply:
I don't believe so